src/azul/service/storage_service.py

```diff
@@ -181,12 +198,16 @@ def upload(self,
 
     def _object_creation_kwargs(self, *,
                                 content_type: str | None = None,
-                                tagging: Tagging | None = None):
+                                tagging: Tagging | None = None,
+                                exists_okay: bool = True
```

Comment:
```suggestion
                                overwrite: bool = True
```

and everywhere else.